pkg/artifactory/resource_artifactory_keypair.go

```diff
@@ -61,15 +61,16 @@ func resourceArtifactoryKeyPair() *schema.Resource {
 				Type:             schema.TypeString,
 				Sensitive:        true,
 				Required:         true,
-				DiffSuppressFunc: stripTabs,
+				DiffSuppressFunc: ignoreEmpty,
 				ValidateDiagFunc: validatePrivateKey,
 				ForceNew:         true,
 			},
 			"passphrase": {
-				Type:        schema.TypeString,
-				Optional:    true,
-				Description: "Used to decrypt the private key (if applicable). Will be verified server side",
-				ForceNew:    true,
+				Type:             schema.TypeString,
+				Optional:         true,
+				DiffSuppressFunc: ignoreEmpty,
+				Description:      "Used to decrypt the private key (if applicable). Will be verified server side",
```

Comment:
We should add to this (and `private_key`) that RT doesn't return the value after creation.

Reply:
added to both